redbot/cogs/mod/settings.py

```diff
@@ -199,3 +202,19 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def defaultdays(self, ctx: commands.Context, days: int = 0):
+        """Set the default days worth of messages to be deleted when a user is banned.
```

Comment:
I think this will be clearer if you add "number of" here.
```suggestion
        """Set the default number of days worth of messages to be deleted when a user is banned.
```